zebrad/tests/acceptance.rs

```diff
@@ -1526,6 +1526,7 @@ async fn fully_synced_rpc_test() -> Result<()> {
 ///
 /// See [`common::lightwalletd::send_transaction_test`] for more information.
 #[tokio::test]
+#[ignore]
```

Comment:
This is good for consistency, we might want to do it for `lightwalletd_wallet_grpc_tests` and `lightwalletd_update_sync` as well.